pos_order_remove_line/static/src/js/orderline.js

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright 2019 LevelPrime
+ *  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+ */
+
+odoo.define("pos_order_remove_line.Orderline", function (require) {
+    "use strict";
+
+    var Orderline = require("point_of_sale.Orderline");
+    const Registries = require("point_of_sale.Registries");
+
+    const PosOrderline = (Orderline) =>
+        class extends Orderline {
+            removeLine() {
+                var line = this.props.line;
+                if (line) {
+                    line.set_quantity("remove");
+                }
```

Comment:
The one liner should be just fine, no?

```suggestion
                this.props.line.set_quantity("remove");
```